nixos/modules/misc/mandoc.nix

```
@@ -36,28 +36,39 @@ in {
           configuration options used for the package.
         '';
       };
+
+      config = lib.mkOption {
```

Typically the name `extraConfig` is used for configuration of type `lines`:

```suggestion
      extraConfig = lib.mkOption {
```